app/views/campaign_pages/_form.slim

```diff
@@ -1,21 +1,24 @@
 = form_for(@campaign_page, class: 'form-horizontal') do |f|
```

Comment:
So, it seems like based on what you'd posted up earlier, we should have a `fields_for` section here that would do fields for all our widgets? 

Is that in line with what you were thinking?